yarn.lock

```diff
@@ -7720,6 +7750,15 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@types/yargs@npm:^17.0.8":
+  version: 17.0.13
+  resolution: "@types/yargs@npm:17.0.13"
+  dependencies:
+    "@types/yargs-parser": "*"
+  checksum: 5005e1e7408b9fea96c356becf19256f1a9bca02120852d3c0089ba7123528b0b6891185e9c92bac25cb5c04090a7a714b201523a6bf4a8a226852205c631208
+  languageName: node
+  linkType: hard
+
```

Comment:
This is the error I'm seeing in team city:
```
Repository contains duplicated packages:

➤ YN0000: ┌ Deduplication step
➤ YN0000: │ @types/yargs@npm:^17.0.1 can be deduped from @types/yargs@npm:17.0.5 to @types/yargs@npm:17.0.13
➤ YN0000: │ One package can be deduped using the highest strategy
➤ YN0000: └ Completed

To fix them, you need to checkout the branch, run 'yarn dedupe',
verify that the new packages work and commit the changes in 'yarn.lock'.
```

Reply:
Thanks for that!